react/features/aeternity/components/TipButton.js

```diff
@@ -259,14 +259,6 @@ class TipButton extends Component<Props, State> {
             return;
         }
 
-        const { value } = this.state;
-
-        if (value[value.length - 1] === '.') {
-            this.setState({ error: 'The last character shouldn\'t be \' . \'' });
-
-            return;
-        }
```

Comment:
I think we can leave that. To stop functions at not valid values is a good practice.
```suggestion
        const { value } = this.state;

        if (value[value.length - 1] === '.') {
            this.setState({ error: 'The last character shouldn\'t be \' . \'' });

            return;
        }
```

Reply:
with the regex fix above this is useless as function will stop before reaching this

Reply:
OK